src/components/Header/Header.tsx

```diff
@@ -0,0 +1,110 @@
+import classNames from 'classnames';
```

Comment:
No need to create a separate folder for the component if there is only 1 file inside

A good example is `TodoItem.tsx` - there is no folder